lib/python/qmk/cli/find.py

```diff
@@ -20,6 +21,7 @@
 def find(cli):
     """Search through all keyboards and keymaps for a given search criteria.
     """
+    os.environ['SKIP_SCHEMA_VALIDATION'] = '1'
```

Comment:
Should perhaps use `.setdefault('SKIP_SCHEMA_VALIDATION', '1')` here, to allow users to override the value. Not that I can think about a good use case to do that, must say.

But also, with the current implementation, a value of `'0'` would not even override.
```py
if not os.environ.get('SKIP_SCHEMA_VALIDATION', None)
# if not '0'
# if not bool('0')
```
Non-empty string evaluates to `True`... even though it was meant as "do not skip"

Reply:
Added a truthy utility function.